scico/test/linop/xray/test_xray_util.py

```diff
@@ -35,6 +47,7 @@ def test_rotate_volume():
     np.testing.assert_allclose(vol.transpose((1, 2, 0)), vol_rot, rtol=1e-7)
 
 
+@pytest.mark.skipif(not have_astra, reason="astra not installed")
```

Comment:
Alternative: https://docs.pytest.org/en/7.1.x/reference/reference.html#pytest-importorskip and remove the import logic at the top of the file.

Reply:
Just to make the benefit clear, `pytest.importorskip` would remove the need for the try except statement at the top of the file.

Reply:
The reason for keeping it as-is is largely for consistency with the practice elsewhere in scico.